jt-colorindexer/src/main/java/it/geosolutions/jaiext/colorindexer/Quantizer.java

```diff
@@ -88,6 +103,9 @@ public ColorIndexer buildColorIndexer(RenderedImage image) {
 
         // setup the first box, that median cut will split in parts
         List<Box> boxes = new ArrayList<Box>();
+        if (histogram.hasTransparentPixels()) {
+            boxes.add(new Box(0, 1, 1, transparentHisto, null));
```

Comment:
Should it update the "colors" count too? I have a hunch the change as is might fail on solid color tiles (e..g, a fully transparent tile)